### PR TITLE
Added GO Sports to sports.json

### PR DESCRIPTION
**Pull request by HussainArif12**
Issue number #3753 https://github.com/osmlab/name-suggestion-index/issues/3753

Added GO Sports to sports.json file

### CODE REVIEW

File: brands/shop/sports.json [hunk 1]
```diff
@@ -446,5 +446,15 @@
       "name:ja": "スーパースポーツゼビオ",
       "shop": "sports"
     }
+  },
+  "shop/sports|GO Sport":{
+    "countryCodes":["fr"],
+    "tags":{
+      "brand": "GO SPORT",
```
Comment (bhousel) [inline brands/shop/sports.json:453]:
```suggestion
      "brand": "GO Sport",
```

File: brands/shop/sports.json [hunk 2]
```diff
@@ -446,5 +446,15 @@
...
+      "brand:wikidata": "Q48746237",
+      "brand:wikipedia":"fr:GO_Sport",
+      "name": "GO SPORT",
```
Comment (bhousel) [inline brands/shop/sports.json:456]:
```suggestion
      "name": "GO Sport",
```

File: brands/shop/sports.json [hunk 3]
```diff
@@ -446,5 +446,15 @@
       "name:ja": "スーパースポーツゼビオ",
       "shop": "sports"
     }
+  },
+  "shop/sports|GO Sport":{
+    "countryCodes":["fr"],
+    "tags":{
+      "brand": "GO SPORT",
+      "brand:wikidata": "Q48746237",
+      "brand:wikipedia":"fr:GO_Sport",
```
Comment (bhousel) [inline brands/shop/sports.json:455]:
```suggestion
      "brand:wikipedia": "fr:GO Sport",
```

**Comment by HussainArif12** [comment]
My apologies. I've fixed it now in the latest commit

**Comment by bhousel** [comment]
Looks good @HussainArif12, thanks for the quick fixes! :+1:

**Pull request merged**